internal/cmd/create.go

```diff
@@ -105,15 +90,24 @@ Examples:
 			for i := range args {
 				switch i {
 				case 0:
+					if opts.Product != "" && opts.Product != args[i] {
+						return errors.New("product can only be specified once")
+					}
 					opts.Product = args[i]
 				case 1:
+					if opts.Vulnerability != "" && opts.Vulnerability != args[i] {
+						return errors.New("vulnerability can only be specified once")
+					}
 					opts.Vulnerability = args[i]
 				case 2:
+					if opts.Status != "" && opts.Status != args[i] {
+						return errors.New("status can only be specified once")
+					}
```

Comment:
This has its origins back when vexctl supported more than one product on `create`, etc. You can quickly define one of them using the params or have more control with the flags. We could change it later if it makes more sense but it is a breaking change and we need to deprecate one of the mechanisms and notify users.

Reply:
Okay, thanks for the context. This certainly doesn't block this PR. It might be worth thinking about a UX "audit" at some point in the future and deciding what deprecations make sense as you think about what your vision is for the CLI.